doc/source/whatsnew/v1.5.0.rst

```diff
@@ -57,6 +57,39 @@ Styler
 
 - Fixed bug in :class:`CSSToExcelConverter` leading to ``TypeError`` when border color provided without border style for ``xlsxwriter`` engine (:issue:`42276`)
 
+.. _whatsnew_150.notable_bug_fixes.groupby_transform_dropna:
+
+Using ``dropna=True`` with ``groupby`` transformers
```

Comment:
do we use "transformer" elsewhere?  Could use "transform"/"transforms"?  Is there a risk of confusion for ML people for whom "transformer" means something else?

Reply:
We do mention transformer once in the docs, but agreed with sticking to transform here.